From 029d0bc423fab0983ac4d4b3deaf6b57327d1ab0 Mon Sep 17 00:00:00 2001
From: Michael Walle <michael@walle.cc>
Date: Wed, 1 Sep 2021 14:18:04 +0200
Subject: [PATCH] arm: dts: ls1028a: move the FlexSPI controller node

While inserting it into the new location, keep it sorted by the
register base offset just like in the linux kernel device tree.

Signed-off-by: Michael Walle <michael@walle.cc>
---
 arch/arm/dts/fsl-ls1028a.dtsi | 26 +++++++++++++-------------
 1 file changed, 13 insertions(+), 13 deletions(-)

diff --git a/arch/arm/dts/fsl-ls1028a.dtsi b/arch/arm/dts/fsl-ls1028a.dtsi
index de85fdd045..6d80b32816 100644
--- a/arch/arm/dts/fsl-ls1028a.dtsi
+++ b/arch/arm/dts/fsl-ls1028a.dtsi
@@ -43,19 +43,6 @@
 					  IRQ_TYPE_LEVEL_LOW)>;
 	};
 
-	fspi: flexspi@20c0000 {
-		compatible = "nxp,lx2160a-fspi";
-		#address-cells = <1>;
-		#size-cells = <0>;
-		reg = <0x0 0x20c0000 0x0 0x10000>,
-		      <0x0 0x20000000 0x0 0x10000000>;
-		reg-names = "fspi_base", "fspi_mmap";
-		clocks = <&clockgen 4 3>, <&clockgen 4 3>;
-		clock-names = "fspi_en", "fspi";
-		interrupts = <GIC_SPI 25 IRQ_TYPE_LEVEL_HIGH>;
-		status = "disabled";
-	};
-
 	serial0: serial@21c0500 {
 		device_type = "serial";
 		compatible = "fsl,ns16550", "ns16550a";
@@ -484,5 +471,18 @@
 			clocks = <&clockgen 4 0>;
 			status = "disabled";
 		};
+
+		fspi: flexspi@20c0000 {
+			compatible = "nxp,lx2160a-fspi";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x0 0x20c0000 0x0 0x10000>,
+			      <0x0 0x20000000 0x0 0x10000000>;
+			reg-names = "fspi_base", "fspi_mmap";
+			clocks = <&clockgen 4 3>, <&clockgen 4 3>;
+			clock-names = "fspi_en", "fspi";
+			interrupts = <GIC_SPI 25 IRQ_TYPE_LEVEL_HIGH>;
+			status = "disabled";
+		};
 	};
 };
-- 
2.30.2

